kotlin/lib/src/main/java/im/vector/app/features/analytics/plan/PinnnedMessageCount.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.analytics.plan
+
+import im.vector.app.features.analytics.itf.VectorAnalyticsEvent
+
+// GENERATED FILE, DO NOT EDIT. FOR MORE INFORMATION VISIT
+// https://github.com/matrix-org/matrix-analytics-events/
+
+/**
+ * Number of pinned messages in a room.
+ */
+data class PinnnedMessageCount(
```

Comment:
Seems a bit strange this is an event and not a property on another event? When does this event fire?

Reply:
We want to know how many pinned messages there is in each room

Reply:
Right that's why in my mind it would be a property on another event. 

What is the guidance on when the clients should send this event? On room join? On room open? other?

Reply:
After discussion with the team, we decided to remove it